File: sakai-ng-master/src/app/Models/MaquillajeViewModel.ts
export class Maquillaje{
    maqu_Id?:String;
    maqu_Nombre?:String;
    maqu_Stock?:String;
    maqu_PrecioCompra?:String;
    maqu_PrecioVenta?:String;
    maqu_PrecioMayor?:String;
    prov_Proveedor?:String;
    maqu_Imagen?:string;
    marc_Marca?:String;
}
export class dropMaqui{
    value?:String;
    text?:String;
}
export class MaquillajeEnviar {
    Maqu_Id?:String;
    Maqu_Nombre: string;
    Maqu_PrecioCompra: string;
    Maqu_PrecioVenta: string;
    Maqu_PrecioMayor: string;
    Maqu_Stock?:String;
    Maqu_Imagen: string;
    Prov_Id: string;
    Marc_Id: string;


}

export class Fill {
    maqu_Id: string;
    maqu_Stock?:String;
    maqu_Nombre: string;
    maqu_PrecioCompra?:String;
    maqu_PrecioVenta?:String;
    maqu_PrecioMayor?:String;
    maqu_Imagen: string;
    marc_Id?:String;
    marc_Marca?:String;
    prov_Id?:String;
    prov_Proveedor?:String;
    usuarioCreacion: string;
    usuarioModificacion: string;
    fechaCreacion : string;
    fechaModificacion : string;
}

File: API Buena/SistemaEsmeralda/SistemaEsmeralda.Common/Models/MaquillajeViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaEsmeralda.Common.Models
{
    public class MaquillajeViewModel
    {
        public int Usua_ID { get; set; }
        public int Maqu_Stock { get; set; }
        public int Maqu_Id { get; set; }
        public string Maqu_Nombre { get; set; }
        public decimal Maqu_PrecioCompra { get; set; }
        public decimal Maqu_PrecioVenta { get; set; }
        public decimal Maqu_PrecioMayor { get; set; }
        public string Maqu_Imagen { get; set; }
        public int Prov_Id { get; set; }
        public int Marc_Id { get; set; }
        public int? Maqu_UsuarioCreacion { get; set; }
        public DateTime? Maqu_FechaCreacion { get; set; }
        public int? Maqu_UsuarioModificacion { get; set; }
        public DateTime? Maqu_FechaModificacion { get; set; }
        public bool? Maqu_Estado { get; set; }
        [NotMapped]
        public string UsuarioCreacion { get; set; }

        [NotMapped]
        public string UsuarioModificacion { get; set; }
    }
}
